toolboxes/feel/feelmodels/levelset/levelsetadvection_inst.cpp

```diff
@@ -0,0 +1,65 @@
+
+#include "levelsetconfig.h"
```

Comment:
use `<>` with the proper path otherwise at installation we will be in trouble

Reply:
Actually, `levelsetconfig.h` is a file generated by CMake from `levelsetconfig.h.in`, so its directory depends on the current build (in particular the instantiation polynomial order, etc.).